Servers/Zeus/ARC/.arc/submit.py

```diff
@@ -0,0 +1,40 @@
+"""
+Submit scripts
+sorted in a dictionary with server names as keys
+"""
+
+
+submit_scripts = {
```

Comment:
I would be great if you could also add the molpro script, as it is coming soon

Reply:
I first want to test it to see that it works... then will upload a working version